track.go

```diff
@@ -61,14 +60,10 @@ func newTrack(opts *MediaDevicesOptions, d driver.Driver, constraints MediaTrack
 	case driver.VideoRecorder:
 		kind = VideoInput
 		rtpCodecs = opts.codecs[webrtc.RTPCodecTypeVideo]
-		buildSampler = newVideoSampler
 		encoderBuilders, err = newVideoEncoderBuilders(r, constraints)
 	case driver.AudioRecorder:
 		kind = AudioInput
 		rtpCodecs = opts.codecs[webrtc.RTPCodecTypeAudio]
-		buildSampler = func(t LocalTrack) samplerFunc {
-			return newAudioSampler(t, constraints.selectedMedia.Latency)
```

Comment:
In my experience, using raw system time, which contains jitter, to RTP timestamp causes problem on playback, and it required low pass filter to use system time based duration. (It might work without visible problem if CPU usage is enough low.)

Reply:
Oh that's good to know! My issue currently is that we don't have a good way to propagate the latency information. We currently got it from the selected property, but the latency itself can be dynamic and changed in transformations, which lead to an incorrect value.

Reply:
Or maybe we can use DetectChanges, https://github.com/pion/mediadevices/blob/master/pkg/io/audio/detect.go#L12, to calculate the latency using the chunk length and sample rate?